client/src/features/add-bet/components/forms/finish-bet-form/finish-bet-inputs/SportInput.tsx

```
@@ -1,27 +1,98 @@
+          disabled={disabled}
+          placeholder="e.g. Premier League"
+        />
+      </div>
+      {showSearch && findSport.trim() && filteredSport.length > 0 && (
+        <div className="sport-search">
+          {filteredSport.map((sport) => (
+            <button
+              key={sport}
+              className="sport-list"
+              onClick={() => handleSelectSuggestion(sport)}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)